test_libs/pyspec/eth2spec/test/context.py

```diff
@@ -8,6 +8,11 @@
 
 from typing import Any, Callable, Sequence
 
+from importlib import reload
+
+reload(spec_phase0)
```

Comment:
Why `reload` here?

Reply:
A comment is probably a good idea, oops. Why: the presets are loaded into another module for the full test run. When spec phase 0 and 1 modules are initialized, they override any of the constants with these presets. And this ensures that the spec module is loaded after the correct configuration is loaded. Result: no re-initialization code in the spec code-gen, just reload and done.